packages/ng/libraries/material/src/style/components/_datepicker.scss

```diff
@@ -6,42 +6,47 @@
 }
 
 .mat-datepicker-toggle {
-	@extend .textfield-suffix;
```

Comment:
Why removing the extend and duplicating the style ?

Reply:
Because we added a pointer-event: none for suffixes (a click on search icon inside a textfield prevented the focus ), but datepicker still need to be clickable.

Reply:
then you might want to still use the extend and then change the pointer-events afterwards. This allows to keep a common base of styling